pandas/io/formats/style.py

```diff
@@ -611,7 +624,9 @@ def _compute(self):
             r = func(self)(*args, **kwargs)
         return r
 
-    def _apply(self, func, axis=0, subset=None, **kwargs):
+    def _apply(
+        self, func: Callable, axis: Optional[Axis] = 0, subset=None, **kwargs
```

Comment:
Can the args or return type of `Callable` be annotated?

Reply:
Only the retrun type (from what I know)

Reply:
Can you add return types then? Still very helpful